src/front-end/components/gallery/Gallery.js

```diff
@@ -3,16 +3,39 @@ import '../app/App.css';
 import CoverComponent from '../cover/Cover';
 import covers from '../covers/Covers';
 
-const Gallery = () => {
-  return (
-    <div className='container'> 
-    {   
-      covers.map(function(cover) {
-        return <CoverComponent id= {cover.id} title={cover.title} image={cover.image} message={cover.message} />;
-      })
+class Gallery extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      covers: []
     }
-    </div>
 
-  );
+  }
+
+  componentDidMount() {
+    fetch('/rest/shows')
+    .then(response => {
+      console.log(response);
+      return response.json();
+    })
+    .then((covers) => { 
+      console.log(covers);
+      this.setState({ covers }); 
+    });
+  }
+
+  render() {
+    return (
+      <div className='container'> 
+      {   
+          covers.map(function(cover) {
+          return <CoverComponent id={cover.id} title={cover.title} image={cover.image} message={cover.message} />;
```

Comment:
Because this line is quite long, can you please split it?
```
return <CoverComponent 
    id={cover.id} 
    title={cover.title} 
    image={cover.image} 
    message={cover.message} />;
```